db/migrate/20160605134652_move_parameters_to_docker_parameters.rb

```diff
@@ -0,0 +1,18 @@
+class MoveParametersToDockerParameters < ActiveRecord::Migration
+  def up
+  # All the  DockerContainerWizardStates::PARAMETER are temporary for the wizard step so no need to keep them
+   docker_params = select_all("select name, value, reference_id, type from parameters where type in ('EnvironmentVariable', 'ForemanDocker::Dns', 'ExposedPort')")
+   docker_params.each do |param|
+     DockerParameter.create(:key => param['name'], :value => param['value'], :reference_id => param['reference_id'], :type => param['type'])
```

Comment:
Becase EnvironmentVariable, ExposedPort and ForemanDocker::Dns exist and I had issues with trying to add FakeLookupKey in migrations. I'll switch it to a fake object and see if it does work in this case.